client/my-sites/plugins/style.scss

```diff
@@ -110,3 +116,186 @@ body.is-section-plugins:not( .is-nav-unification ) {
+}
+
+.single-plugin__top-section {
+	border-bottom: 1px solid var( --studio-gray-5 );
+
+	.single-plugin__layout-col-left.no-cta {
+		@include grid-column( 1, 12 );
+	}
+
+	.single-plugin__layout-col-right.no-cta {
+		display: none;
+	}
+}
+
+.single-plugin__header {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	height: 400px;
+
+	.single-plugin__name, .single-plugin__price {
+		font-family: Recoleta;
+		font-size: $font-title-large;
+		color: var( --studio-gray-100 );
+		padding: 16px 0;
+	}
+
+	.single-plugin__description {
+		font-size: $font-title-small;
+		color: var( --studio-gray-70 );
+		padding-bottom: 30px;
+	}
+
+	.single-plugin__additional-info {
+		th {
+			font-size: $font-body-extra-small;
+			font-weight: 400;
+			color: var( --studio-gray-40 );
+		}
+
+		td {
+			font-size: $font-body;
+			color: var( --studio-gray-70 );
+		}
+	}
+
+	.single-plugin__install {
+		padding-bottom: 20px;
+
+		.single-plugin__install-button {
+			width: 100%;
+			padding: 12px 16px;
+			background-color: var( --studio-pink-50 );
+			color: var( --studio-white );
+		}
+	}
+	
+	.single-plugin__t-and-c {
+		font-size: $font-body-extra-small;
+		color: var( --studio-gray-50 );
+
+		@media screen and ( max-width: 1040px ) {
+			padding-bottom: 40px;
+		}
+	}
+
+	@media screen and ( max-width: 1040px ) {
+		height: 100%;
+	}
+}
+
+.single-plugin__body {
+	.single-plugin__plugin-details-title {
+		padding: 15px 0;
+		display: none;
+		color: var( --studio-gray-100 );
+		font-size: $font-body-small;
+	}
+
+	.plugin-sections {
+		margin-top: 20px;
+
+		.section-nav {
+			box-shadow: none;
+		}
+
+		.card {
+			box-shadow: none;
+		}
+	}
+
+	.section-nav-tab {
+		&.is-selected,
+		&:hover:not( .is-selected ) {
+			border-bottom-color: transparent;
+		}
+
+		.section-nav-tab__link {
+			font-size: $font-body-small;
+			color: var( --studio-gray-60 );
+
+			&:hover {
+				background-color: transparent;
+				color: var( --studio-black );
+			}
+		}
+
+		&.is-selected .section-nav-tab__link {
+			color: var( --studio-black );
+			font-weight: bold;
+			border-bottom: none;
+		}
+	}
+
+	.single-plugin__layout-col-right {
+		margin-top: 100px;
+
+		.title {
+			color: var( --studio-gray-40 );
+			font-size: $font-body-extra-small;
+		}
+
+		.value {
+			color: var( --studio-gray-70 );
+			font-size: $font-body-small;
+			padding-bottom: 16px;
+
+			&.single-plugin__downloads-value {
+				font-size: $font-body;
+			}
+		}
+	}
+
+	@media screen and ( max-width: 1040px ) {
+		display: flex;
+		flex-direction: column;
+
+		.single-plugin__plugin-details-content {
+			display: flex;
+		}
+
+		.single-plugin__plugin-details-title {
+			display: block;
+		}
+		
+		.single-plugin__layout-col-left {
+			order: 2;
+		}
+		
+		.single-plugin__layout-col-right {
+			order: 1;
+			margin-top: 0;
+			padding: 0 24px;
+			display: flex;
+			flex-direction: column;
+			border-bottom: 1px solid var( --studio-gray-5 );
+		}
+	
+		.single-plugin__version {
```

Comment:
Let's add the version in the header too

![SS 2021-11-09 at 18 17 12](https://user-images.githubusercontent.com/12430020/140962393-a6681aa5-313e-4cc9-bc27-027dbbe0d8f2.png)

Reply:
Sure, added.